### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by rodit** [rodit/SnapMod#92] [Announcements]
For any non-specific issues or discussions (i.e. how to do something, asking if something is supported, feature suggestions etc.), please open a discussion here rather than on the issues tab.

**Comment by flawnn** [comment]
Hey dude, any way to contact you privately? Love your project, just wanna have a chat with you! @rodit

**Reply by rodit** [reply]
You can contact me on discord rodit#0978 but I can't promise to reply in any kind of timely fashion.

**Reply by ghost** [reply]
@rodit https://www.youtube.com/watch?v=0t7D3_Ffppg

don't trust them snap employees lurk and will fuck you over by pressing you to work for them or get sued.

**Reply by flawnn** [reply]
> @rodit https://www.youtube.com/watch?v=0t7D3_Ffppg
> 
> don't trust them snap employees lurk and will fuck you over by pressing you to work for them or get sued.

dude, I ain't a fed or snap employee 😂
but true words, reducing contact options and snap will have more work to do to track shit down if something like that happens

**Reply by authorisation** [reply]
> You can contact me on discord rodit#0978 but I can't promise to reply in any kind of timely fashion.

I added you
